Kitodo/src/main/java/org/kitodo/constants/StringConstants.java

```diff
@@ -15,5 +15,6 @@ public class StringConstants {
 
     public static final String COMMA_DELIMITER = ", ";
     public static final String SEMICOLON_DELIMITER = "; ";
+    public static final String SAVE = "editForm:save";
```

Comment:
Is `SAVE` not to meaningless and may be used in wrong context? Maybe `EDITFORM_SAVE` can prevent this?

Reply:
Yes, you are right. The name of the constant should match the content. When it's called just "SAVE", I, too, would expect it to contain a simple String "Save". I will change that to your suggestion.